doc/Manuals/LoadBalancing/ig-lb_x-road_external_load_balancer_installation_guide.md

```diff
@@ -0,0 +1,691 @@
+| ------------------- | -------------------- | -------------------------------------------------- |
+| serverconf database | **replication required** | PostgreSQL streaming replication (Hot standby) |
+
+The serverconf database replication is done using streaming replication with hot standby. Note that PostgreSQL replication
+is all-or-nothing, it is not possible exclude databases from the replication. This is why the replicated serverconf and
+non-replicated messagelog databases need to be separated to different instances.
+
+
+#### 2.3.2 `messagelog` database replication
+| Data               | Replication          | Replication method                                 |
+| ------------------- | -------------------- | -------------------------------------------------- |
+| messagelog database | **not replicated** |                                                      |
+
+The messagelog database is not replicated. Each node has its own separate messagelog database. **However**, in order to
+support PostgreSQL streaming replication (hot standby mode) for the serverconf data, the serverconf and messagelog
+databases must be separated. This requires modifications to the installation (a separate PostgreSQL instance is needed
+for the messagelog database) and has some implications on the security server resource requirements as since a separate
+instance uses some memory.
+
+#### 2.3.3 Key configuration and software token replication from `/etc/xroad/signer/*`
+| Data                           | Replication          | Replication method                                 |
+| ------------------------------- | -------------------- | -------------------------------------------------- |
+| keyconf and the software token  | **replicated**       |  `rsync+ssh`  (scheduled)                          |
+
+Previously, any external modification to `/etc/xroad/signer/keyconf.xml` was overwritten by the X-Road signer process if
+it was running. Therefore, replicating the signer configuration without service disruptions would have required taking the
+cluster members offline one-by-one. The load balancing support adds the possibility for external modifications to the
+keyconf.xml to be applied on slave nodes without service disruptions. The actual state replication is done with a scheduled
+rsync over ssh. This might take a few minutes so a slight delay in propagating the changes must be tolerated by the
+clustered environment. A small delay should usually cause no problems as new keys and certificates are unlikely to be used
+immediately for X-Road messaging. Changes to the configuration are also usually relatively infrequent. These were one of
+the [basic assumptions](#basic_assumptions) about the environment. Users should make sure this holds true for them.
+
+The slave nodes use the `keyconf.xml` in read-only mode, no changes are persisted to disk. Slaves reload the configuration
```

Comment:
> "no changes are persisted to disk"

 is an explanation to "read-only mode" and should not be separated by just a comma